chatkit-core/src/main/kotlin/com/pusher/chatkit/cursors/CursorSubscriptionEventParser.kt

```diff
@@ -31,8 +32,8 @@ internal object CursorSubscriptionEventParser : DataParser<CursorSubscriptionEve
 
     private fun JsonObject.parseEvent(eventName: String): Result<CursorSubscriptionEvent, Error> =
             when (eventName) {
-                "new_cursor" -> parseAs<Cursor>().map(CursorSubscriptionEvent::OnCursorSet)
-                "cursor_set" -> CursorSubscriptionEvent.NoEvent.asSuccess()
+                "new_cursor" -> parseAs<CursorApiType>().map(CursorSubscriptionEvent::OnCursorSet)
+                "cursor_set" -> parseAs<CursorApiType>().map(CursorSubscriptionEvent::OnCursorSet)
```

Comment:
confirmed with the backend team that cursor_set is legacy, should be removed. indeed now after the recent chagne it would generate duplicated events also on our side

Reply:
Thanks for clarifying with the backend team. Have removed 👍